File: DoubtClass-1(Warm-upExercise-1)/CumulativeSum.cpp
vector<int> getCumulativeSum(vector<int> &arr)
{
    // add your logic here
    int n = arr.size();
    vector<int> v;       // size of vector is 0
    v.push_back(arr[0]); // size of vector 1 v[0] = arr[0]

    for (int i = 1; i < n; i++)
    {
        int curr = arr[i] + v[i - 1];
        v.push_back(curr);
    }

    return v;
}

File: DoubtClass-1(Warm-upExercise-1)/CumulativeSum.java
class Solution {
	int[] getCumulativeSum (int[] arr) {
		// add your logic here
		// for(int i=1;i<arr.length;i++)
		// 	arr[i] += arr[i-1];
		// return arr;
		int n = arr.length;
		int p[] = new int[n];
		p[0] = arr[0];
		for(int i=1;i<n;i++) {
			p[i] = arr[i] + p[i-1];
		}
		return p;
	}
}